smoke/IotEdgeQuickstart/Program.cs

```diff
@@ -48,6 +48,7 @@ be used.
                             Sockets, otherwise N/A
                             switch form uses local IP address as hostname
   --username                anonymous, or Key Vault if --registry is specified
+  --no-deployment           Start IoT Edge modules deployment (EdgeHub and custom modules)
```

Comment:
Nit: Should the help text say something like: `Deploys only Edge Agent if set`.

Reply:
This section is about the default behaviors when a given flag is _not_ set.